host/tests/helpers/dsAudioTestHelper.py

```diff
@@ -0,0 +1,184 @@
+
+        Args:
+            path (str): The file path to the YAML file.
+
+        Returns:
+            dict: The parsed YAML content.
+        """
+        with open(path, 'r') as file:
+            return yaml.safe_load(file)
+
+    def dsAudioInitialise(self):
+        """
+        Initializes the device settings Audio module.
+
+        Args:
+            None.
+
+        Returns:
+            None
+        """
+        self.UT.run_tests(run_script, self.menu_yaml["dsAudio"]["control"]["menu"]["groups"]["name"],
+                          self.menu_yaml["dsAudio"]["control"]["menu"]["groups"]["menu_initialize"]["name"])
+
+    def dsAudioEnablePort(self):
+        """
+        Enables the audio port.
+
+        Args:
+            None.
+
+        Returns:
+            None
+        """
+        output = self.writeCommandOnDevice("s", "Enter number of suite to select")
+
+        index = self.UT.find_index_in_output(output, self.menu_yaml["dsAudio"]["control"]["menu"]["groups"]["menu_enable"]["name"])
+        input = self.menu_yaml["dsAudio"]["control"]["menu"]["groups"]["menu_enable"]["input"]
+        key, value = input.items()
+        output = self.writeCommandOnDevice(str(index), key)
+
+        index = self.UT.find_index_in_output(output, value)
+        output = self.writeCommandOnDevice(str(index), "Enter command")
+
+    def dsAudioVerifyAudio(self, manual=True):
+        """
+        Verifies whether the audio is fine or not.
+
+        Args:
+            None.
+
+        Returns:
+            bool
+        """
+        if manual == True:
+            print("Is Audio playing fine[y\n]:")
+            pass_res = input()
+            if pass_res == 'y' or pass_res == 'Y' :
+                self.log.testResultMessage("Passed")
+                return False
+            else:
+                self.log.testResultMessage("Failed")
+                return True
+        else :
+            #TOD: Add automation verification methods
+            return True
+
+    def dsAudioDisablePort(self):
+        """
+        Enables the audio port.
+
+        Args:
+            None.
+
+        Returns:
+            None
+        """
+        output = self.writeCommandOnDevice("s", "Enter number of suite to select")
+
+        index = self.UT.find_index_in_output(output, self.menu_yaml["dsAudio"]["control"]["menu"]["groups"]["menu_disable"]["name"])
+        input = self.menu_yaml["dsAudio"]["control"]["menu"]["groups"]["menu_disable"]["input"]
+        key, value = input.items()
+        output = self.writeCommandOnDevice(str(index), key)
+
+        index = self.UT.find_index_in_output(output, value)
+        output = self.writeCommandOnDevice(str(index), "Enter command")
+
+    def dsAudioTerminate(self):
+        """
+        Enables the audio port.
+
+        Args:
+            None.
+
+        Returns:
+            None
+        """
+        output = self.writeCommandOnDevice("s", "Enter number of suite to select")
```

Comment:
should use be a command in `ut-raft` to support menuSelection( "L3 dsAudio - Sink", "Terminate dsAudio" ); but I like your idea here with `menuSelection( "menu_terminate" );` - but the problem is how can you generate that automatically from `ut-core`, or should we just be saying we're going to generate these manually?